src/sources/http_server.rs

```diff
@@ -496,16 +612,18 @@ mod tests {
             let log = event.as_log();
             assert_eq!(log[log_schema().message_key()], "test body".into());
             assert!(log.get(log_schema().timestamp_key()).is_some());
-            assert_eq!(log[log_schema().source_type_key()], "http".into());
```

Comment:
👍 good catch, I didn't notice these when I was renaming the source - another reason to use the `NAME` instead!